migration/sql-files/004-add-capacity-exhausted-to-cluster.sql

```diff
@@ -0,0 +1,2 @@
+-- Store capacity_exhausted for cluster
+ALTER TABLE cluster ADD COLUMN capacity_exhausted boolean;
```

Comment:
maybe add `DEFAULT false` to the column definition, for existing clusters in the DB?